rocks/tempest/rockcraft.yaml

```diff
@@ -0,0 +1,44 @@
+name: tempest
+summary: OpenStack integration test suite
+license: Apache-2.0
+description: |
+  Tempest is a set of integration tests to be run against a live OpenStack
+  cluster. Tempest has batteries of tests for OpenStack API validation,
+  Scenarios, and other specific tests useful in validating an OpenStack
+  deployment. This ROCK is built using the snap-tempest.
+version: '2023.2'
+
+# renovate: base: ubuntu:22.04@sha256:2b7412e6465c3c7fc5bb21d3e6f1917c167358449fecac8176c6e496e5c1f05f
+base: ubuntu@22.04
+
+platforms:
+  amd64:
+
+parts:
+  snap-tempest:
+    plugin: python # we need to use python plugin because we need a python interpreter to run tempest internally
+    source: .
+    stage-snaps:
+    - tempest/2023.2/stable
+    stage-packages:
+    - cron
+    - python3-venv # this is required for python plugin
+  tempest-user:
```

Comment:
Can you move tempest-user first, and have snap-tempest depend on it ?

That's what's done is other Rocks because we want to create the user before the overlay packages do. But here it does not really, so it would be better if it looked the same as the others.

Reply:
Updated

Reply:
Thank you!